src/components/molecules/SpecialMenu/SpecialMenu.module.scss

```diff
@@ -0,0 +1,25 @@
+@use "sass:map";
+@import "../../../foundations/sass/all";
+
+.special-menu {
+  &__header {
+    color: $special-menu-header-color;
+    font-size: $fs-xxl;
+    font-weight: $fw-bold;
+    text-align: center;
+    text-transform: uppercase;
+
+    @include margin-bottom("xxl");
+  }
+
+  &__meals-container {
+    flex-wrap: wrap;
+    gap: map.get($spacing, md);
+
+    @include center;
+
+    & > * {
```

Comment:
Why are you using this selector? I haven't seen a need for it. Using this selector causes side effects. One component should not affect the style of other components.